lib/billy/capybara/capybara.rb

```diff
@@ -0,0 +1,52 @@
+require 'billy'
+
+module Billy
```

Comment:
I'd consider making a folder called `drivers` instead of `capybara` and then adding a module here called `Drivers::Capybara`:

```ruby
module Billy
  module Drivers::Capybara
   
    def self.register_drivers
      # ...
     end
  end
end
```

At first glance having a capybara specific folder looks like you're going to monkey patch it. And being explicit with the modules means you don't have the problem with `Billy.register_drivers` already existing.